tokens/repository-map.json

```diff
@@ -2,75 +2,96 @@
     "acute-care": [
         "automated-interpretation-pipeline",
         "fewgenomes",
-        "gatk-sv"
+        "gatk-sv",
+        "rare-disease"
     ],
     "ag-hidden": [
-        "automated-interpretation-pipeline"
+        "automated-interpretation-pipeline",
+        "rare-disease"
     ],
     "ancestry": [
         "ancestry"
     ],
     "bioheart": [],
     "brain-malf": [
-        "automated-interpretation-pipeline"
+        "automated-interpretation-pipeline",
+        "rare-disease"
     ],
     "circa": [
-        "automated-interpretation-pipeline"
+        "automated-interpretation-pipeline",
+        "rare-disease"
     ],
     "common": [],
     "constraint": [
         "structural-constraint"
     ],
     "epileptic-enceph": [
-        "automated-interpretation-pipeline"
+        "automated-interpretation-pipeline",
+        "rare-disease"
     ],
     "fewgenomes": [
         "automated-interpretation-pipeline",
         "fewgenomes",
         "images",
         "joint-calling",
         "production-pipelines",
+        "rare-disease",
         "sv-workflows"
     ],
     "flinders-ophthal": [],
-    "gtex": [],
+    "gtex": [
+        "rare-disease"
```

Comment:
Is gtex a RD project?

Reply:
It is a reference dataset that RD projects will make extensive use of.